specs/pvs/corla.pvs

```diff
@@ -1133,6 +1216,13 @@ BEGIN
   random_list_of_ballots:
     [set[ballot] -> sequence[ballot]]
 
+% @trace Per email from Jerome Lovato at CDOS on 19 July 2017: The
+% tool must also provide a printable list of ballot for each round.
+% At a minimum, the list must contain ballot style and ballot manifest
+% information.
+  print_ballots_under_audit_list:
+    [sequence[ballot] -> ballots_under_audit_instructions]
+
 % @trace RDPR-6-Jul-2017 Section 25.2.2(K)
 
 %%% \subsection{Random Selection of Ballots for Audit}
```

Comment:
line 1141/1231 This sounds as if SoS needs to provide a system for pseudo-random number generation, when in fact the RLA tool provides the pseudo-random number generation. All the SoS provides is the seed. Clarify.